help/view-and-browse-images.md

```diff
@@ -1,36 +1,83 @@
 # View and browse images
 
 When someone pastes or attaches an image to a message, Zulip shows a small
-preview. Click on the image preview to open the **image viewer**.
+preview. You can open the image viewer to see an image in more detail or
+download it.
 
-In the image viewer, you can:
+## Using the image viewer
 
-* Zoom in and out of the image
+{start_tabs}
 
-* Click and drag the image
+{tab|desktop-web}
 
-* **Reset zoom** so that the image is recentered and its original size
+1. Click on the image preview to open the **image viewer**.
 
-* **Open** the image in a new browser tab
+1. Interact with the image.
```

Comment:
Maybe add: "You can:"? We aren't really telling users to do *all* of these things every time.

Reply:
(same goes for mobile)